### PR TITLE
[WIP] Move error strings into a file

### LINKED ISSUE #5264: Move error strings into a separate file
Start using approved error strings to log errors. 

1. Move all existing error strings into a separate file.
2. Log `errorCode`.
3. Log `Tags`

### PR DESCRIPTION
**Pull request by kannappanr**
Moves all the error strings to a single file. Each error string
is contained within a corresponding error logging function.

Fixes #5264

## How Has This Been Tested?

Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/log-errors.go [hunk 1]
```diff
@@ -0,0 +1,1061 @@
+package cmd
```
Comment (harshavardhana) [inline cmd/log-errors.go:1]: Add license header, we should move the logger to a new package along with errorIf and fatalIf.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/log-errors.go [hunk 2]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogFetchDiskInfoFailed logs failure to fetch disk info.
+func LogFetchDiskInfoFailed(err error, disk string) {
+	m := make(map[string]string)
+	m["Disk"] = disk
+	errorIf(err, funcName(), "Unable to fetch disk info.", m)
+}
+
+// test-utils_test.go
```
Comment (vadmeste) [inline cmd/log-errors.go:1051]: I don't think we need to include *_test.go stuffs in this file.

File: cmd/log-errors.go [hunk 3]
```diff
@@ -0,0 +1,1061 @@
...
+// LogConfigMigrationFailed logs failure to migrate config.
+func LogConfigMigrationFailed(err error) {
+	fatalIf(err, funcName(), "Config migration failed.", nil)
+}
+
+// LogFailedconfigLoad logs failure to load config.
+func LogFailedconfigLoad(err error, version string) {
+	m := make(map[string]string)
+	m["ExpectedVersion"] = version
+	fatalIf(err, funcName(), "Unable to load config.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:291]: `"Unable to load config."` => `"Unable to load server config."`

File: cmd/log-errors.go [hunk 4]
```diff
@@ -0,0 +1,1061 @@
...
+		return functionNamePath[len(functionNamePath)-1]
+	}
+	return ""
+}
+
+// admin-handlers.go
+
+// LogUptimeFailure logs Uptime update Failure from majority of servers.
+func LogUptimeFailure(err error) {
+	errorIf(err, funcName(), "Possibly failed to get uptime from majority of servers.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:22]: `"Possibly failed to get uptime from majority of servers."` => `"Failed to get up-time from majority of servers."`

File: cmd/log-errors.go [hunk 5]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Failed to unmarshal JSON configuration.", nil)
+}
+
+//admin-rpc-server.go
+
+// LogFailedWriteServerTempConfig logs failure to write temporary config file on the server.
+func LogFailedWriteServerTempConfig(err error, path string) {
+	m := make(map[string]string)
+	m["ConfigPath"] = path
+	errorIf(err, funcName(), "Failed to write temporary config file.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:95]: `"Failed to write temporary config file."` => `"Failed to write to server temporary config file."`

File: cmd/log-errors.go [hunk 6]
```diff
@@ -0,0 +1,1061 @@
...
+
+// admin-rpc-client.go
+
+// LogFailedWriteTempConfig logs failure to write temporary config file.
+func LogFailedWriteTempConfig(err error) {
+	errorIf(err, funcName(), "Failed to write temporary config file.", nil)
+}
+
+// LogFailedConfigRename logs failure to rename config file.
+func LogFailedConfigRename(err error, source, destination string) {
```
Comment (ebozduman) [inline cmd/log-errors.go:106]: `LogFailedConfigRename` => `LogFailedRenameConfig`

File: cmd/log-errors.go [hunk 7]
```diff
@@ -0,0 +1,1061 @@
...
+	m := make(map[string]string)
+	m["sourceConfig"] = source
+	m["destinationConfig"] = destination
+	errorIf(err, funcName(), "Failed to rename config file.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:110]: `"Failed to rename config file."` => `"Failed to rename local config file."`

File: cmd/log-errors.go [hunk 8]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogFailedRemoteConfigRename logs failure to rename config file on a remote node.
+func LogFailedRemoteConfigRename(err error) {
```
Comment (ebozduman) [inline cmd/log-errors.go:114]: `LogFailedRemoteConfigRename` => `LogFailedRenameRemoteConfig`

File: cmd/log-errors.go [hunk 9]
```diff
@@ -0,0 +1,1061 @@
...
+	m := make(map[string]string)
+	m["ConfigPath"] = path
+	errorIf(err, funcName(), "Failed to write temporary config file.", m)
+}
+
+// admin-rpc-client.go
+
+// LogFailedWriteTempConfig logs failure to write temporary config file.
+func LogFailedWriteTempConfig(err error) {
+	errorIf(err, funcName(), "Failed to write temporary config file.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:102]: `"Failed to write temporary config file."` => `"Failed to write to local temporary config file."`

File: cmd/log-errors.go [hunk 10]
```diff
@@ -0,0 +1,1061 @@
...
+func LogFailedConfigRename(err error, source, destination string) {
+	m := make(map[string]string)
+	m["sourceConfig"] = source
+	m["destinationConfig"] = destination
+	errorIf(err, funcName(), "Failed to rename config file.", m)
+}
+
+// LogFailedRemoteConfigRename logs failure to rename config file on a remote node.
+func LogFailedRemoteConfigRename(err error) {
+	errorIf(err, funcName(), "Failed to rename config file on a remote node.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:115]: `"Failed to rename config file on a remote node."` => `"Failed to rename remote config file."`

File: cmd/log-errors.go [hunk 11]
```diff
@@ -0,0 +1,1061 @@
...
+// LogFailedFetchUptime logs failure to fetch uptime
+func LogFailedFetchUptime(err error) {
+	errorIf(err, funcName(), "Unable to fetch uptime.", nil)
+}
+
+// LogFailedUnMarshalPeerServerConfig logs failure to unmarshal serverConfig.
+func LogFailedUnMarshalPeerServerConfig(err error, addr string) {
+	m := make(map[string]string)
+	m["PeerAddress"] = addr
+	errorIf(err, funcName(), "Failed to unmarshal serverConfig.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:127]: `"Failed to unmarshal serverConfig."` => `"Failed to unmarshal server config."`

File: cmd/log-errors.go [hunk 12]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogFailedFindServerConfig logs failure to finad a valid server config."
+func LogFailedFindServerConfig(err error) {
+	errorIf(err, funcName(), "Unable to finad a valid server config.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:132]: `"Unable to finad a valid server config."` => `"Unable to find a valid server config."`

File: cmd/log-errors.go [hunk 13]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogInvalidHTTPReqHeader logs invalid http request header was found.
+func LogInvalidHTTPReqHeader(err error) {
+	errorIf(err, funcName(), "found invalid http request header", nil)
+}
+
+// LogFailedInitHashReader logs failure to initialize hashReader.",
+func LogFailedInitHashReader(err error) {
+	errorIf(err, funcName(), "Unable to initialize hashReader.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:183]: `"Unable to initialize hashReader."` => `"Unable to initialize hash reader."`

File: cmd/log-errors.go [hunk 14]
```diff
@@ -0,0 +1,1061 @@
...
+	fatalIf(err, funcName(), "Unable to get home directory.", nil)
+}
+
+// net.go
+
+// LogUnexpectedErrorResolvingHost logs failure to resolve host"
+func LogUnexpectedErrorResolvingHost(err error, host string) {
+	m := make(map[string]string)
+	m["Host"] = host
+	fatalIf(err, funcName(), "unexpected error when resolving host.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:360]: `"unexpected error when resolving host."` => `"Unexpected error when resolving host."`

File: cmd/log-errors.go [hunk 15]
```diff
@@ -0,0 +1,1061 @@
...
+func LogFailedInitListenerTargetLogger(err error) {
+	errorIf(err, funcName(), "Unable to initialize listener target logger.", nil)
+}
+
+// format-xl.go
+
+// LogErrorDiskNotFound logs that a disk was not found in JBOD list.
+func LogErrorDiskNotFound(err error, uuid string) {
+	m := make(map[string]string)
+	m["UUID"] = uuid
```
Comment (ebozduman) [inline cmd/log-errors.go:419]: `"UUID"` => `"DiskID"`

File: cmd/log-errors.go [hunk 16]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Disk not found in JBOD list.", m)
+}
+
+// LogDiskWrongOrder logs that disk is in wrong order.
+func LogDiskWrongOrder(err error, uuid string, expectedIndex, index int) {
+	m := make(map[string]string)
+	m["UUID"] = uuid
```
Comment (ebozduman) [inline cmd/log-errors.go:426]: "UUID" => "DiskID"

File: cmd/log-errors.go [hunk 17]
```diff
@@ -0,0 +1,1061 @@
...
+	m["ReceivedOrder"] = strconv.Itoa(index)
+	errorIf(err, funcName(), "Disk is in wrong order.", m)
+}
+
+// fs-v1-multipart.go
+
+// LogFailedAccessUploadsJSON logs failure to access uploads.json.
+func LogFailedAccessUploadsJSON(err error, uploadIDPath string) {
+	m := make(map[string]string)
+	m["UploadIdsPath"] = uploadIDPath
```
Comment (ebozduman) [inline cmd/log-errors.go:437]: "UploadIdsPath"` => `"UploadsIdPath"`

File: cmd/log-errors.go [hunk 18]
```diff
@@ -0,0 +1,1061 @@
...
+// LogFailedReadObject logs failure to read object.
+func LogFailedReadObject(err error, bucket, object string) {
+	m := make(map[string]string)
+	m["BucketName"] = bucket
+	m["ObjectName"] = object
+	errorIf(err, funcName(), "Unable to read object.", m)
+}
+
+// LogFailedRemoveFile logs failure to remove file.
+func LogFailedRemoveFile(err error, path, basePath string) {
```
Comment (ebozduman) [inline cmd/log-errors.go:450]: `"path"` is defined as `pathJoin(basePath, object)` , so it already has the `"basePath"` information in it. Hence, there is no point printing `"basePath"` information as part of error `"variables"`.

`"Unable to remove file."` => `"Unable to remove object."`

We can get rid of it in the function definition and the call:
```
func LogFailedRemoveFile(err error, path string) {
   m := make(map[string]string)
   m["Path"] = path
   errorIf(err, funcName(), "Unable to remove object.", m)
}

```

and on line#1014 in `"cmd/fs-v1-multipart.go"`:
```
LogFailedRemoveFile(derr, pathJoin(basePath, object))
```

File: cmd/log-errors.go [hunk 19]
```diff
@@ -0,0 +1,1061 @@
...
+
+// LogResourceLimitChangeFailed logs failure to change resource limit.
+func LogResourceLimitChangeFailed(err error) {
+	errorIf(err, funcName(), "Unable to change resource limit.", nil)
+}
+
+// LogInvalidAddressInCli logs invalid address in CLI.
+func LogInvalidAddressInCli(err error, addr string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:546]: `"Addr"` => `"ServerAddress"`

File: cmd/log-errors.go [hunk 20]
```diff
@@ -0,0 +1,1061 @@
...
+	fatalIf(err, funcName(), "Invalid address in command line argument", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:547]: `"Invalid address in command line argument"` => `"Invalid server address in command line argument"`

File: cmd/log-errors.go [hunk 21]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogInvalidArgsInCli logs invalid args in CLI.
+func LogInvalidArgsInCli(err error, addr, args string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:553]: `"Addr"` => `"ServerAddress"`

File: cmd/log-errors.go [hunk 22]
```diff
@@ -0,0 +1,1061 @@
...
+	m := make(map[string]string)
+	m["UploadID"] = uploadID
+	errorIf(err, funcName(), "unable to remove meta data object for upload ID.", m)
+}
+
+// gateway-gcs.go
+
+// LogGCSMissingProjectID logs and erro message that project-id should be provided as argument or GOOGLE_APPLICATION_CREDENTIALS should be set with path to credentials.json.
+func LogGCSMissingProjectID(err error) {
+	errorIf(err, funcName(), "project-id should be provided as argument or GOOGLE_APPLICATION_CREDENTIALS should be set with path to credentials.json.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:603]: ```"project-id should be provided as argument or GOOGLE_APPLICATION_CREDENTIALS should be set with path to credentials.json."```
 => 
```"project-id should be provided as argument or GOOGLE_APPLICATION_CREDENTIALS should be set with path in credentials.json."```

File: cmd/log-errors.go [hunk 23]
```diff
@@ -0,0 +1,1061 @@
...
+	m := make(map[string]string)
+	m["ProjectID"] = projectID
+	errorIf(err, funcName(), "Unable to start GCS gateway.", m)
+}
+
+// LogGCSObjectListPurgeError logs failure to list objects during purging of old files in minio.sys.tmp.
+func LogGCSObjectListPurgeError(err error, bucket string) {
+	m := make(map[string]string)
+	m["BucketName"] = bucket
+	errorIf(err, funcName(), "Object listing error during purging of old files in minio.sys.tmp.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:617]: `"Object listing error during purging of old files in minio.sys.tmp."` => `"Object listing error on bucket during purging of old files in minio.sys.tmp."`

File: cmd/log-errors.go [hunk 24]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogGCSDeleteObjectPurgeError logs failure to delete object during purging of old files in minio.sys.tmp
+func LogGCSDeleteObjectPurgeError(err error, bucket, attribute string) {
+	m := make(map[string]string)
+	m["BucketName"] = bucket
+	m["Attribute"] = attribute
```
Comment (ebozduman) [inline cmd/log-errors.go:624]: `"Attribute"` => `"ObjectName"`

File: cmd/log-errors.go [hunk 25]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// gateway-sia.go
+
+// LogMissingSiaFilePath logs failure in finding file uploaded to Sia path.
+func LogMissingSiaFilePath(err error, bucket, object string) {
+	m := make(map[string]string)
+	m["BucketName"] = bucket
+	m["ObjectName"] = object
+	errorIf(err, funcName(), "Unable to find file uploaded to Sia path.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:640]: `"Unable to find file uploaded to Sia path."` => `"Unable to find object uploaded to Sia path."`

File: cmd/log-errors.go [hunk 26]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Failed to clear the lock state.", m)
+}
+
+// LogFailedDeleteLockInfoEntryForOpsID logs failure to delete lock info entry.
+func LogFailedDeleteLockInfoEntryForOpsID(err error, volume, path, opsID string) {
+	m := make(map[string]string)
+	m["Volume"] = volume
+	m["Path"] = path
+	m["OpsID"] = opsID
+	errorIf(err, funcName(), "Failed to delete lock info entry.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:710]: `"Failed to delete lock info entry."` => `"Failed to delete lock info entry for operation."`

File: cmd/log-errors.go [hunk 27]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Failed to set the lock state to running.", nil)
+}
+
+// LogInvalidReferenceCount logs invalid reference count detected
+func LogInvalidReferenceCount(err error) {
+	errorIf(err, funcName(), "Invalid reference count detected.", nil)
+}
+
+// LogFailedDeleteLockInfoEntry logs failure to delete lock info entry.
+func LogFailedDeleteLockInfoEntry(err error) {
```
Comment (ebozduman) [inline cmd/log-errors.go:732]: We don't need this function. 
It is the exact same function `"LogFailedDeleteLockInfoEntryForVolumePath()"` created for `"deleteLockInfoEntryForVolumePath()"` action the same way.

Removing this function also requires the following changes:
in Line#240 of `cmd/namespace-lock.go`:
`LogFailedDeleteLockInfoEntry(err)` => `LogFailedDeleteLockInfoEntryForOpsID(err, param.volume, param.path, opsID)`

in Line#251 of `cmd/namespace-lock.go`:
`LogFailedDeleteLockInfoEntry(err)` => `LogFailedDeleteLockInfoEntryForVolumePath(err, param.volume, param.path)`

File: cmd/log-errors.go [hunk 28]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Failed to delete lock info entry for volume path.", nil)
+}
+
+// net-rpc-client.go
+
+// LogFailedSecureConnection logs failure to establish secure connection"
+func LogFailedSecureConnection(err error, addr string) {
+	m := make(map[string]string)
+	m["Address"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:741]: `"Address"` => `"ServerAddress"`

File: cmd/log-errors.go [hunk 29]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Failed to delete lock info entry for volume path.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:742]: `"Failed to delete lock info entry for volume path."` => `"Unable to establish secure connection to server."`

File: cmd/log-errors.go [hunk 30]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogESConnectFailed logs failure to connect to elastic search service.
+func LogESConnectFailed(err error) {
+	errorIf(err, funcName(), "Unable to connect to elastic search service.", nil)
+}
+
+// LogPostGreSQLConnectFailed logs failure to connect to PostGreSQL Server.
+func LogPostGreSQLConnectFailed(err error) {
+	errorIf(err, funcName(), "Unable to connect to PostGreSQL service.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:796]: `"Unable to connect to PostGreSQL service."` => `"Unable to connect to PostgreSQL server."`

File: cmd/log-errors.go [hunk 31]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogMySQLConnectFailed logs failure to connect to MySQL server.
+func LogMySQLConnectFailed(err error) {
+	errorIf(err, funcName(), "Unable to connect to MySQL server.", nil)
+}
+
+// LogKafkaDialFailed logs failure to dial kafka server.
+func LogKafkaDialFailed(err error) {
+	errorIf(err, funcName(), "Unable to dial kafka server.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:806]: `"Unable to dial kafka server."` => `"Unable to dial Kafka server."`

File: cmd/log-errors.go [hunk 32]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// notify-webhook.go
+
+// LogLookupWebhookFailed logs failure to lookup webhook endpoint.
+func LogLookupWebhookFailed(err error, urlStr string) {
+	m := make(map[string]string)
+	m["URL"] = urlStr
+	errorIf(err, funcName(), "Unable to lookup webhook endpoint.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:815]: `"Unable to lookup webhook endpoint."` => `"Unable to lookup WebHook endpoint."`

File: cmd/log-errors.go [hunk 33]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogFailedParseFormValues logs failure to parse form values.
+func LogFailedParseFormValues(err error) {
+	errorIf(err, funcName(), "Unable to parse form values.", nil)
+}
+
+// LogInvalidHTTPReqHeader logs invalid http request header was found.
+func LogInvalidHTTPReqHeader(err error) {
+	errorIf(err, funcName(), "found invalid http request header", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:178]: `"found invalid http request header"` => `"Invalid http request header"`

File: cmd/log-errors.go [hunk 34]
```diff
@@ -0,0 +1,1061 @@
...
+	m := make(map[string]string)
+	m["Disk"] = disk
+	errorIf(err, funcName(), "Disk is still unreachable.", m)
+}
+
+// LogDiskWrongFormat logs disk detected in unexpected format.
+func LogDiskWrongFormat(err error, disk string) {
+	m := make(map[string]string)
+	m["Disk"] = disk
+	errorIf(err, funcName(), "Disk detected in unexpected format.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:881]: `"Disk detected in unexpected format."` => `"Unexpected format on disk."`

File: cmd/log-errors.go [hunk 35]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// rpc-server.go
+
+// LogRPCHijackingFailed logs failure of rpc hijacking.
+func LogRPCHijackingFailed(err error, remoteAddr string) {
+	m := make(map[string]string)
+	m["RemoteAddr"] = remoteAddr
+	errorIf(err, funcName(), "rpc hijacking failed.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:890]: `"rpc hijacking failed."` => `"Failure during HTTP/RPC hijack."`

File: cmd/log-errors.go [hunk 36]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+//s3-peer-client.go
+
+// LogPeerIndexOutofBounds logs peerIndex out of bounds"
+func LogPeerIndexOutofBounds(err error, index int) {
+	m := make(map[string]string)
+	m["Index"] = strconv.Itoa(index)
+	errorIf(err, funcName(), "peerIndex out of bounds.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:899]: `"peerIndex out of bounds."` => `"Peer index out of bounds."`

File: cmd/log-errors.go [hunk 37]
```diff
@@ -0,0 +1,1061 @@
...
+	m["RemoteAddr"] = remoteAddr
+	errorIf(err, funcName(), "rpc hijacking failed.", m)
+}
+
+//s3-peer-client.go
+
+// LogPeerIndexOutofBounds logs peerIndex out of bounds"
+func LogPeerIndexOutofBounds(err error, index int) {
+	m := make(map[string]string)
+	m["Index"] = strconv.Itoa(index)
```
Comment (ebozduman) [inline cmd/log-errors.go:898]: `"Index"` => `"PeerIndex"`

File: cmd/log-errors.go [hunk 38]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "peerIndex out of bounds.", m)
+}
+
+// LogUpdateBucketNotificationError logs failure to send update of bucket notification.
+func LogUpdateBucketNotificationError(err error, addr string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:905]: `"Addr"` => `"PeerAddress"`

File: cmd/log-errors.go [hunk 39]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Error sending update bucket notification.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:906]: `"Error sending update bucket notification."` => `"Error sending update bucket notification to peer."`

File: cmd/log-errors.go [hunk 40]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// LogUpdateBucketListenerError logs failure to sending update bucket listener.
+func LogUpdateBucketListenerError(err error, addr string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:912]: `"Addr"` => `"PeerAddress"`

File: cmd/log-errors.go [hunk 41]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Error sending update bucket listener.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:913]: `"Error sending update bucket listener."` => `"Error sending update bucket listener to peer."`

File: cmd/log-errors.go [hunk 42]
```diff
@@ -0,0 +1,1061 @@
...
+
+// LogFailedPolicyChangeMarshal logs failure to marshal policyChange",
+func LogFailedPolicyChangeMarshal(err error) {
+	errorIf(err, funcName(), "Failed to marshal policyChange.", nil)
+}
+
+// LogUpdateBucketPolicyError logs failure sending update bucket policy"
+func LogUpdateBucketPolicyError(err error, addr string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:924]: `"Addr"` => `"PeerAddress"`

File: cmd/log-errors.go [hunk 43]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Error sending update bucket policy.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:925]: `"Error sending update bucket policy."` => `"Error sending update bucket policy to peer."`

File: cmd/log-errors.go [hunk 44]
```diff
@@ -0,0 +1,1061 @@
...
+
+// LogBOSHCheckError logs error in BOSH check.
+func LogBOSHCheckError(err error) {
+	errorIf(err, funcName(), "Error in BOSH check.", nil)
+}
+
+// LogReadHelmFileFailed logs failure to read helm info file.
+func LogReadHelmFileFailed(err error, helmPath string) {
+	m := make(map[string]string)
+	m["HelmPath"] = helmPath
```
Comment (ebozduman) [inline cmd/log-errors.go:972]: `"HelmPath"` => `"HelmFilePath"`

File: cmd/log-errors.go [hunk 45]
```diff
@@ -0,0 +1,1061 @@
...
+	errorIf(err, funcName(), "Unable to read helm info file.", m)
+}
+
+// utils.go
+
+// LogHTTPTraceCloseFailed logs failure to close httpTraceFile.
+func LogHTTPTraceCloseFailed(err error, httpTrace string) {
+	m := make(map[string]string)
+	m["HttpTraceFile"] = httpTrace
+	errorIf(err, funcName(), "Unable to close httpTraceFile.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:982]: `"Unable to close httpTraceFile."` => `"Unable to close http trace file."`

File: cmd/log-errors.go [hunk 46]
```diff
@@ -0,0 +1,1061 @@
...
+}
+
+// web-handlers.go
+
+// LogRemoteLoginReqFailed logs failure to login request from remote.
+func LogRemoteLoginReqFailed(err error, addr string) {
+	m := make(map[string]string)
+	m["Addr"] = addr
```
Comment (ebozduman) [inline cmd/log-errors.go:990]: `"Addr"` => `"RemoteAddress"`

File: cmd/log-errors.go [hunk 47]
```diff
@@ -0,0 +1,1061 @@
...
+	m["Prefix"] = prefix
+	m["XLMetaJSON"] = xlMetaJSON
+	errorIf(err, funcName(), "Unable to stat file.", m)
+}
+
+// xl-v1-utils.go
+
+// LogInternalDiskEvaluateFailed logs failure to evaluate internal disks.
+func LogInternalDiskEvaluateFailed(err error) {
+	errorIf(err, funcName(), "unable to evaluate internal disks.", nil)
```
Comment (ebozduman) [inline cmd/log-errors.go:1019]: `"unable to evaluate internal disks."` => `"Unable to evaluate internal disks."`

File: cmd/log-errors.go [hunk 48]
```diff
@@ -0,0 +1,1061 @@
...
+// LogGetMaxCacheSizeFailed logs failure to get maximum cache size.
+func LogGetMaxCacheSizeFailed(err error) {
+	errorIf(err, funcName(), "Unable to get maximum cache size.", nil)
+}
+
+// LogFetchDiskInfoFailed logs failure to fetch disk info.
+func LogFetchDiskInfoFailed(err error, disk string) {
+	m := make(map[string]string)
+	m["Disk"] = disk
+	errorIf(err, funcName(), "Unable to fetch disk info.", m)
```
Comment (ebozduman) [inline cmd/log-errors.go:1048]: `"Unable to fetch disk info."` => `"Unable to fetch info for disk."`

**Review by ebozduman — CHANGES_REQUESTED** [review]

**Comment by kannappanr** [comment]
Not needed anymore. Closing this PR.

**Pull request closed without merging**